RestaurantReservation.Db/Customers.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RestaurantReservation.Db
+{
+    public class Customers
+    {
+        public int CustomersId { get; set; }
```

Comment:
what is the purpose of plural? can't it be just **CustomerId** without "**s**", the same applies for other properties in other tables, pls check them also